src/neural/network_cudnn.cu

```diff
@@ -1001,7 +1000,53 @@ class CudnnNetwork : public Network {
 
     // 1. allocate scratch space (used internally by cudnn to run convolutions,
     //     and also for format/layout conversion for weights)
-    reportCUDAErrors(cudaMalloc(&scratch_mem_, kCudaScratchSize));
+    cudnnFilterDescriptor_t wDesc;
```

Comment:
The variable name style should be `w_desc`.
But as there are lots of variables with different styles sneaked into the file already, this can be done in a separate PR (probably I'll do that).